cmd/controller-manager/app/controller_manager.go

```diff
@@ -34,9 +35,13 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/tools/record"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/apiserver/pkg/server/healthz"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
```

Comment:
you might want to cc @deads2k or other parts of @kubernetes/sig-api-machinery-misc about that